waku/persistence/store.go

```diff
@@ -392,19 +404,41 @@ func (d *DBStore) Query(query *pb.HistoryQuery) (*pb.Index, []StoredMessage, err
 		orderDirection = "DESC"
 	}
 
+	d.log.Info(fmt.Sprintf("conditionstr: %s", conditionStr))
+	d.log.Info(fmt.Sprintf("parameters: %s", parameters))
+
 	paramCnt++
+	d.log.Info(fmt.Sprintf("paramCnt: %d", paramCnt))
+
 	sqlQuery += fmt.Sprintf("LIMIT $%d", paramCnt)
 	sqlQuery = fmt.Sprintf(sqlQuery, conditionStr, orderDirection, orderDirection, orderDirection, orderDirection)
+	d.log.Info(fmt.Sprintf("sqlQuery: %s", sqlQuery))
```

Comment:
I'd change all these logs from lines 407 to 415, to `Debug` if they're really necessary, otherwise we can remove them. Also the logger uses zap, so instead of using `fmt.Sprintf`, you should be able to do something like:
```
d.log.Debug("some message",  zap.String("conditionStr", conditionStr), zap.String("parameters", parameters), etc etc) 
```

Reply:
Oh man, missed removing these debug statements i added for debugging some issue i ran into. Will remove them.